Dockerfile-alpine.template

```
@@ -91,9 +91,30 @@ COPY php.ini /usr/local/etc/php/conf.d/php-matomo.ini
 
 COPY docker-entrypoint.sh /entrypoint.sh
 
-# WORKDIR is /var/www/html (inherited via "FROM php")
-# "/entrypoint.sh" will populate it at container startup from /usr/src/matomo
-VOLUME /var/www/html
+ENV APACHE_DOCUMENT_ROOT /var/www/matomo
+WORKDIR ${APACHE_DOCUMENT_ROOT}
+
+RUN \
+  mv /usr/src/matomo/config /usr/src/config && \
+  mv /usr/src/matomo/plugins /usr/src/plugins && \
+  mv /usr/src/matomo/* ${APACHE_DOCUMENT_ROOT}
+
+# Fix permissions
+RUN \
+  chgrp -R 0 . && \
```

```suggestion
  chgrp -R 0 ${APACHE_DOCUMENT_ROOT} && \
```

Also I think those 3 RUNs could easily be combined as they probably run quite fast.

---

I've combined them now.